lib/matobj2.gd

```diff
@@ -130,6 +130,10 @@
 ##  However, row list matrices behave nearly like lists of row vectors
 ##  that insist on being dense and containing only vectors of the same
 ##  length and with the same base domain.
+##  <P/>
+##  New <Ref Filt="IsMatrixObj"/> and <Ref Filt="IsVectorObj"/> representations
+##  have to implement methods for the operations listed in Section
+##  <Ref Sect="Implementing New Vector and Matrix Objects Types"/>
```

Comment:
```suggestion
##  <Ref Sect="Implementing New Vector and Matrix Objects Types"/>.
```

Reply:
Hmm, but of course this is just the minimum; in practice some more will almost always be implemented for performance? But anyway, shouldn't this be explained in detail in section "Implementing New Vector and Matrix Objects Types"; and then we just put a link to that here?

Reply:
Jepp, I've replaced this by only a link to that section.

Reply:
Will you still apply my fix (adding a missing `.`)?

Reply:
Done.